src/Client/Updater.php

```diff
@@ -590,10 +590,14 @@ public function download(string $target, ...$extra): PromiseInterface
     {
         $this->refresh();
 
-        // @todo Handle the possibility that the target's metadata might not be
-        // in targets.json.
-        // @see https://github.com/php-tuf/php-tuf/issues/116
-        $targetsMetaData = TargetsMetadata::createFromJson($this->durableStorage['targets.json']);
+        $snapShotMetadata = SnapshotMetadata::createFromJson($this->durableStorage['snapshot.json']);
```

Comment:
Nit: Capitalization inconsistency between "targetsMetaData" and "snapShotMetadata". IMHO this should simply be "targetsMetadata" and "snapshotMetadata".

Reply:
fix was fixed in another PR